src/components/popup/PopupService.js

```diff
@@ -45,8 +45,17 @@ goog.require('ga_draggable_directive');
         $(document.body).append(this.element);
       };
 
-      Popup.prototype.open = function() {
+      Popup.prototype.open = function(autoclose) {
+        var that = this;
         this.scope.toggle = true;
+        if (autoclose) {
+          $timeout(function() {
+            if (that && that.scope && that.scope.toggle) {
+              that.close();
+            }
+          }, autoclose);
```

Comment:
Can we change the `autoclose` parameter to `autocloseDelay` . Without I had the feeling this varianble was a boolean.